shared/admin-tools/dynamo/get-cases-by-docket-numbers.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable max-lines */
```

Comment:
was this script meant to be a part of this PR?

Reply:
looks like it's from the temp `9589-bug` branch

Reply:
Not sure why this is still showing up on the diff even after staging has been merged to test. I blame Github

Reply:
ah gotcha!

Reply:
...I think you were right the first time: `9589-bug` _was_ merged into this branch for some reason. Will remove